pyccl/halos/profiles.py

```diff
@@ -919,21 +919,48 @@ class HaloProfileHernquist(HaloProfile):
     Args:
         c_M_relation (:obj:`Concentration`): concentration-mass
             relation to use with this profile.
+        fourier_analytic (bool): set to `True` if you want to compute
+            the Fourier profile analytically (and not through FFTLog).
+            Default: `False`.
+        projected_analytic (bool): set to `True` if you want to
+            compute the 2D projected profile analytically (and not
+            through FFTLog). Default: `False`.
+        cumul2d_analytic (bool): set to `True` if you want to
+            compute the 2D cumulative surface density analytically
+            (and not through FFTLog). Default: `False`.
         truncated (bool): set to `True` if the profile should be
             truncated at :math:`r = R_\\Delta` (i.e. zero at larger
             radii.
     """
     name = 'Hernquist'
 
-    def __init__(self, c_M_relation, truncated=True):
+    def __init__(self, c_M_relation,
+                 fourier_analytic=False,
+                 projected_analytic=False,
+                 cumul2d_analytic=False,
```

Comment:
Can you preserve the order of `truncated`? otherwise, if someone had a code that did
```
HaloProfileHernquist(cM, False)
```
it will silently not do what they're thinking! (thus breaking the API)

Reply:
I see. Fixed.